frontend/src/components/Common/ReportDialog/reportDoc.tsx

```diff
@@ -11,20 +12,33 @@ import { Theme } from '@material-ui/core';
 import { TableRow as AnalysisTableRow } from 'context/analysisResultStateSlice';
 import { getLegendItemLabel } from 'components/MapView/utils';
 import { LegendDefinition, ReportType } from 'config/types';
-import { TFunction } from 'utils/data-utils';
 import { Column } from 'utils/analysis-utils';
+import RobotoFont from 'fonts/Roboto-Medium.ttf';
```

Comment:
Looks like Roboto-Regular.tff is added but never used?

Reply:
Robot is used conditionally below

Reply:
yes but only the medium one, no? not the regular file that you also committed to the repo

Reply:
Yes you are right I wanted actually to add the Regular so the Medium is not necessary sorry for the misconception, those were a part of some testing I'll fix it